source/comp/markv_codec.cpp

```diff
@@ -290,6 +292,8 @@ class CommentLogger {
   }
 
   void AppendBitSequence(const std::string& str) {
+    if (debug_consumer_)
```

Comment:
Today I learned that a std::function is convertible to bool, and its meaning.    :-)